packages/core-components/src/layout/SignInPage/guestProvider.tsx

```diff
@@ -21,35 +21,38 @@ import { InfoCard } from '../InfoCard/InfoCard';
 import { GridItem } from './styles';
 import { ProviderComponent, ProviderLoader, SignInProvider } from './types';
 import { GuestUserIdentity } from './GuestUserIdentity';
+import { coreComponentsTranslationRef } from '../../translation';
+import { useTranslationRef } from '@backstage/core-plugin-api/alpha';
 
-const Component: ProviderComponent = ({ onSignInStarted, onSignInSuccess }) => (
-  <GridItem>
-    <InfoCard
-      title="Guest"
-      variant="fullHeight"
-      actions={
-        <Button
-          color="primary"
-          variant="outlined"
-          onClick={() => {
-            onSignInStarted();
-            onSignInSuccess(new GuestUserIdentity());
-          }}
-        >
-          Enter
-        </Button>
-      }
-    >
-      <Typography variant="body1">
-        Enter as a Guest User.
-        <br />
-        You will not have a verified identity,
-        <br />
-        meaning some features might be unavailable.
-      </Typography>
-    </InfoCard>
-  </GridItem>
-);
+const Component: ProviderComponent = ({ onSignInStarted, onSignInSuccess }) => {
+  const { t } = useTranslationRef(coreComponentsTranslationRef);
+  return (
+    <GridItem>
+      <InfoCard
+        title={t('signIn.guestProvider.title')}
+        variant="fullHeight"
+        actions={
+          <Button
+            color="primary"
+            variant="outlined"
+            onClick={() => {
+              onSignInStarted();
+              onSignInSuccess(new GuestUserIdentity());
+            }}
+          >
+            {t('signIn.guestProvider.enter')}
+          </Button>
+        }
+      >
+        <Typography variant="body1">
+          {t('signIn.guestProvider.subtitle')}
+          <br />
+          {t('signIn.guestProvider.description')}
```

Comment:
Think we can condense these two into one key with a `\n` or `<br/>` in the middle whichever one works.

Reply:
done